src/test/java/HomepageTest.java

```diff
@@ -1,45 +1,137 @@
+        SoftAssertions softAssertions = new SoftAssertions();
+        softAssertions.assertThat(homepage.isErinBannerDisplayed()).isTrue();
+        softAssertions.assertThat(homepage.isErinLinkEnabled()).isTrue();
+        softAssertions.assertThat(homepage.getErinBannerTopText()).isEqualTo("Take it from Erin");
+        softAssertions.assertThat(homepage.getErinBannerMainText()).isEqualTo("Luma founder Erin Renny shares her favorites!");
+        softAssertions.assertThat(homepage.getErinBannerBottomText()).isEqualTo("Shop Erin Recommends");
+        softAssertions.assertThat(homepage.getErinBannerLink()).isEqualTo(HOMEPAGE_URL + "collections/erin-recommends.html");
+        softAssertions.assertThat(homepage.getErinBannerImage()).isEqualTo(HOMEPAGE_URL + "pub/media/wysiwyg/home/home-erin.jpg");
+        softAssertions.assertAll();
+    }
+
+    @Test
+    public void homepage_EcoFriendlyBanner_HasTextAndLinkCorrectlyDisplayed() {
+        //then
+        SoftAssertions softAssertions = new SoftAssertions();
+        softAssertions.assertThat(homepage.isEcoFriendlyBannerDisplayed()).isTrue();
+        softAssertions.assertThat(homepage.isEcoFriendlyLinkEnabled()).isTrue();
+        softAssertions.assertThat(homepage.getEcoFriendlyBannerTopText()).isEqualTo("Twice around, twice as nice");
+        softAssertions.assertThat(homepage.getEcoFriendlyBannerMainText()).isEqualTo("Find conscientious, comfy clothing in our eco-friendly collection");
+        softAssertions.assertThat(homepage.getEcoFriendlyBannerBottomText()).isEqualTo("Shop Eco-Friendly");
+        softAssertions.assertThat(homepage.getEcoFriendlyBannerLink()).isEqualTo(HOMEPAGE_URL + "collections/eco-friendly.html");
+        softAssertions.assertThat(homepage.getEcoFriendlyBannerImage()).isEqualTo(HOMEPAGE_URL + "pub/media/wysiwyg/home/home-eco.jpg");
+        softAssertions.assertAll();
+    }
+
+    @Test
+    public void homepage_PerformanceBanner_HasTextAndLinkCorrectlyDisplayed() {
+        //then
+        SoftAssertions softAssertions = new SoftAssertions();
+        softAssertions.assertThat(homepage.isPerformanceBannerDisplayed()).isTrue();
+        softAssertions.assertThat(homepage.isPerformanceLinkEnabled()).isTrue();
+        softAssertions.assertThat(homepage.getPerformanceBannerTopText()).isEqualTo("Science meets performance");
+        softAssertions.assertThat(homepage.getPerformanceBannerMainText()).isEqualTo("Wicking to raingear, Luma covers you");
+        softAssertions.assertThat(homepage.getPerformanceBannerBottomText()).isEqualTo("Shop Performance");
+        softAssertions.assertThat(homepage.getPerformanceBannerLink()).isEqualTo(HOMEPAGE_URL + "collections/performance-fabrics.html");
+        softAssertions.assertThat(homepage.getPerformanceBannerImage()).isEqualTo(HOMEPAGE_URL + "pub/media/wysiwyg/home/home-performance.jpg");
+        softAssertions.assertAll();
+    }
+
+    @Test
+    public void homepage_HotSellers_HasTextAndLinkCorrectlyDisplayed() {
```

Comment:
Invalid test method name. It's doesn't check any link, only presence of products, their images, names and details section.